Network/Messages/UDPPacket/UDPPacket.cpp

```diff
@@ -12,7 +12,7 @@ namespace ABINetwork
 
 ABINetwork::UDPPacket::UDPPacket(const std::string& packetData)
 {
-    if (packetData.size() < 15)
+    if (packetData.size() < 14)
```

Comment:
What is this 14 ? + It should be a define

Reply:
You're right, it's the minimum length of a payload.

Reply:
Okay 👌